Formula/trino.rb

```diff
@@ -0,0 +1,75 @@
+class Trino < Formula
+  desc "Distributed SQL query engine for big data"
+  homepage "https://trino.io"
+  url "https://search.maven.org/remotecontent?filepath=io/trino/trino-server/360/trino-server-360.tar.gz"
+  sha256 "cd313867fd5461a26920ca6cb170413893853b0cc1acc4f454d4141a0790a50e"
+  license "Apache-2.0"
+
+  livecheck do
+    url "https://search.maven.org/remotecontent?filepath=io/trino/trino-server/"
+    regex(%r{href=["']?v?(\d+(?:\.\d+)*)/?["' >]}i)
+  end
+
+  depends_on "gnu-tar" => :build
+  depends_on arch: :x86_64
+  depends_on "openjdk"
+
+  resource "trino-src" do
+    url "https://github.com/trinodb/trino/archive/360.tar.gz", using: :nounzip
+    sha256 "48fbe2f29f67b5c655d4de6eaec905c94f735c42df17e5509dc3c387e732aeb2"
+  end
+
+  resource "trino-cli" do
+    url "https://search.maven.org/remotecontent?filepath=io/trino/trino-cli/360/trino-cli-360-executable.jar"
+    sha256 "9203abd2ee3e36b7476aba23102b93e2adca7b030031fa3828e2320cb4c38be2"
+  end
+
+  def install
+    libexec.install Dir["*"]
+
+    # Manually untar, since macOS-bundled tar produces the error:
+    #   trino-360/plugin/trino-hive/src/test/resources/<truncated>.snappy.orc.crc: Failed to restore metadata
+    # Remove when https://github.com/trinodb/trino/issues/8877 is fixed
+    resource("trino-src").stage do |r|
+      system "gtar", "-xzf", "trino-#{r.version}.tar.gz"
```

Comment:
`gtar` is macOS-specific. Let's do this instead:
```suggestion
      ENV.prepend_path Formula["gnu-tar"].opt_libexec/"gnubin"
      system "tar", "-xzf", "trino-#{r.version}.tar.gz"
```